sklearn/tests/test_dummy.py

```diff
@@ -564,13 +571,14 @@ def test_most_frequent_strategy_sparse_target():
                                 [1, 0]]))
 
     n_samples = len(X)
-    clf = DummyClassifier(strategy="most_frequent", random_state=0)
-    clf.fit(X, y)
-
-    y_pred = clf.predict(X)
-    assert_true(sp.issparse(y_pred))
-    assert_array_equal(y_pred.toarray(), np.hstack([np.ones((n_samples, 1)),
-                                                    np.zeros((n_samples, 1))]))
+    proba = np.hstack([np.ones((n_samples, 1)), np.zeros((n_samples, 1))])
```

Comment:
I don't understand why this variable is named "proba". It should probably better be named:

`expected_predictions` or `y_expected`.